frontend/src/game/board/builder.js

```diff
@@ -40,6 +42,12 @@ module.exports = {
         newCard.classList.remove("col-1");
         newCard.classList.add("col-2");
         newCard.innerHTML = cardData["power"];
+        newCard.addEventListener('click', function() {
+            if (status.clickedCard !== undefined &&
+                status.clickedCard.hasAttribute("row-played")) {
+                play.playCard(true)
```

Comment:
wut

Reply:
Si apreto una carta en joc i ja tinc una carta seleccionada i jugada (és a dir, l'hi he assignat quina és la seva `row-played` jugo la `status.clickedCard`. 
El `true` ara per ara és per indicar que ja s'ha seleccionat el target. Podria ser qualsevol valor que no sigui `undefined`. Quan fem la issue que passa el target, aquest valor serà el target.

Reply:
then i feel this `true` should be in a variable, and that variable would be set to true for now